doc/src/quick_start_guide.md

```diff
@@ -50,13 +52,13 @@ lalrpop_mod!(grammar);
 
 [lalrpop_mod]: https://docs.rs/lalrpop-util/latest/lalrpop_util/macro.lalrpop_mod.html
 
-#### Running manually
+## Running manually
 
 If you prefer, you can also run the `lalrpop` crate as an
 executable. Simply run `cargo install lalrpop` and then you will get a
 `lalrpop` binary you can execute, like so:
 
-```
+```bash
```

Comment:
Usually I believe `console` is preferred for command lines, while `bash` is more for bash files (although it can be fall back, because `console` isn't always supported - but I'm pretty sure it's supporter on GH and major platforms and renderers)

```suggestion
```console
```